### PR TITLE
Clean up typography on job descriptions

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Clean up typography on job descriptions by removing inline styles and empty paragraphs.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click through different job descriptions
- Ensure typography is consistent

### CODE REVIEW

**Comment by clagom** [comment]
lgtm @barrymcgee thanks

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Suggestion when resetting the headings.

**Comment by anthonydillon** [comment]
Works well in the QA :+1:

File: static/sass/styles.scss [hunk 1]
```diff
@@ -187,3 +187,22 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
...
+.job-desc {
+  h2,
+  h3 {
+    font-size: 1.7rem;
+    font-style: normal;
+
+    strong {
+      font-weight: 100 !important;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (barrymcgee) [reply static/sass/styles.scss:207]: Ok, done!

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**